tensorflow/examples/udacity/.gitignore

```diff
@@ -0,0 +1,5 @@
+notMNIST.pickle
```

Comment:
It's an intended edit :)
Changed data_folders to check for folders rather than somewhat flimsy check for .DS_Store.